tests/rule_based_profiler/data_assistant/test_column_value_missing_data_assistant.py

```diff
@@ -87,3 +90,45 @@ def test_column_value_missing_data_assistant_plot_expectations_and_metrics_corre
     # This test passes only if absense of any metrics and expectations to plot does not cause exceptions to be raised.
     column_domain_charts: List[dict] = [p.to_dict() for p in plot_result.charts[2:]]
     assert len(column_domain_charts) == 0
+
+
+@pytest.mark.integration
+@pytest.mark.slow
+def test_single_batch_multiple_columns(empty_data_context):
+    context = empty_data_context
+    datasource = context.sources.add_or_update_pandas("my_datasource")
+    asset = datasource.add_dataframe_asset("my_asset")
+    df = pd.DataFrame(
+        {
+            "non-null": [i for i in range(100)],
+            "null": [None for _ in range(100)],
+            "low-null": [None for _ in range(38)] + [i for i in range(62)],
+        }
+    )
+    batch_request = asset.build_batch_request(dataframe=df)
+
+    context.assistants._register("my_data_assistant", ColumnValueMissingDataAssistant)
+    result = context.assistants.my_data_assistant.run(batch_request=batch_request)
+
+    assert len(result.expectation_configurations) == 3
+
+    expected_results = {
+        "null": {"mostly": 1.0, "expectation": "expect_column_values_to_be_null"},
+        "non-null": {
+            "mostly": 1.0,
+            "expectation": "expect_column_values_to_not_be_null",
+        },
+        "low-null": {
+            "mostly": 0.6,
+            "expectation": "expect_column_values_to_not_be_null",
+        },
+    }
+
+    for expectation in result.expectation_configurations:
+        assert expectation.expectation_type in [
+            "expect_column_values_to_not_be_null",
+            "expect_column_values_to_be_null",
+        ]
+        column = expectation.kwargs["column"]
```

Comment:
@TrangPham We have a practice of adding type hints.  The documented style is to add it when it is hard to tell what it would be (so, `column: str` for this case).  I tend to err on the side of adding them, rather than skipping them (arguing consistency, even when it slightly contravenes the style guide).  Hence, please look over your code in this PR and decide what you will do.  Thanks!

Reply:
Thanks! I'll update the tests to ensure that they have the same standard of typing as our main code.

Reply:
🙇